.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: MCWertGaming, Alekuso
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
Since the 15th line has 3 dots, I'd also put 3 dots for the 16th and 17th lines, not 4

Reply:
LOL, those are official github templates xD Will fix that quickly.